config/Config-kernel.in

```diff
@@ -1184,9 +1194,56 @@ config KERNEL_SECURITY_SELINUX_SID2STR_CACHE_SIZE
 	depends on KERNEL_SECURITY_SELINUX
 	default 256
 
+config KERNEL_SECURITY_APPARMOR
+	bool "AppArmor Support"
+	select KERNEL_AUDIT
+	select KERNEL_SECURITY
+	select KERNEL_SECURITY_FS
+	select KERNEL_SECURITY_PATH
+	select KERNEL_SECURITY_NETWORK
+
+config KERNEL_DEFAULT_SECURITY_APPARMOR
+	bool "Select AppArmor as default security module"
+
+config KERNEL_SECURITY_APPARMOR_BOOTPARAM_VALUE
+	int
```

Comment:
How is a user supposed to change this if there is no text added? This item won't appear in the menu.